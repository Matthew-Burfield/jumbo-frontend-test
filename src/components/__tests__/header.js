import React from 'react'
import { render } from '@testing-library/react'
import Header from '../header'
describe('Header', () => {
  it('renders correctly', () => {
    const { container } = render(<Header siteTitle="Default Starter" />)
    expect(container).toMatchSnapshot()
  })
})
