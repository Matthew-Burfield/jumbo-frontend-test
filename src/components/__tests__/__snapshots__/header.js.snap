// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <header
    class="css-h21gvl-Header"
  >
    <svg
      fill="none"
      height="59"
      width="66"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M59.57 55.14c3.84 0 6.43-2.61 6.43-6.48V6.48C66 2.62 63.4 0 59.57 0H6.43C2.59 0 0 2.62 0 6.48V59l3.3-3.86V6.48c0-1.74 1.4-3.15 3.13-3.15h53.14a3.15 3.15 0 0 1 3.13 3.15v42.18c0 1.74-1.4 3.15-3.13 3.16H11.32l-3.3 3.32-.02-.02"
        fill="#01D277"
        fill-rule="evenodd"
      />
      <path
        clip-rule="evenodd"
        d="M16.14 20h1.72v-7.21H20V11h-6v1.79h2.14V20zM26.28 20H28v-9h-1.72v3.6h-2.56V11H22v9h1.72v-3.6h2.56V20zM35 18.2h-3.26v-1.8h2.78v-1.8h-2.78v-1.8h3.11V11H30v9h5v-1.8zM18 26.9L14.57 23H14v9h1.77v-4.95l2.24 2.38 2.23-2.38V32H22v-9h-.55l-3.44 3.9zM28.5 23c-6 0-6 9 0 9s6-9 0-9zm0 7.2c-3.49 0-3.49-5.4 0-5.4s3.49 5.4 0 5.4z"
        fill="#01D277"
        fill-rule="evenodd"
      />
      <path
        d="M43 23h2v9h-2z"
        fill="#01D277"
      />
      <path
        clip-rule="evenodd"
        d="M48.74 30.2v-1.8h2.78v-1.8h-2.78v-1.8h3.11V23H47v9h5v-1.8h-3.26zM14 35h2.62c5.84 0 5.84 9 0 9H14v-9zm1.76 7.2h.86c3.41 0 3.41-5.4 0-5.4h-.86v5.4zM28.1 39.5c.55-.39.78-1.1.8-1.77.04-1.59-.92-2.73-2.46-2.73H23v9h3.44c.68 0 1.34-.3 1.82-.8s.75-1.2.74-1.9a2.2 2.2 0 0 0-.9-1.8zm-3.35-2.7h1.54a.9.9 0 0 1 .81.9.9.9 0 0 1-.8.9h-1.55v-1.8zm0 5.4h1.54c.24 0 .46-.09.62-.26.17-.17.26-.4.26-.64a.9.9 0 0 0-.26-.64.84.84 0 0 0-.62-.26h-1.54v1.8zM37.5 27.76L35.1 23H33l4.3 9h.4l4.3-9h-2.1l-2.4 4.76z"
        fill="#01D277"
        fill-rule="evenodd"
      />
    </svg>
    <div
      class="css-14o0pyv"
    />
    <div
      class="css-3lrhva"
    />
    <div
      class="css-1ysrk0y"
    />
    <div
      class="css-b717fi"
    />
    <div
      class="css-1dce5hk"
    />
    <div
      class="css-1nia92h"
    />
  </header>
</div>
`;
